src/breadcrumb/breadcrumb.scss

```
@@ -0,0 +1,54 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-breadcrumb {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  align-items: center;
+
+  > .iui-button {
+    margin-right: $iui-s;
+  }
+  
+  ol {
+    @include iui-reset;
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    user-select: none;
+
+    li {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      max-width: $iui-3xl * 2;
+
+      a:not([aria-current]) {
+        @include iui-anchor;
+      }
+
+      [aria-current] {
+        text-decoration: none;
+        @include iui-focus;
+        @include themed {
+          color: t(iui-text-color);
+        }
+      }
+
+      &:not(:first-child)::before {
+        content: '\203A';
```

Maybe leave a comment next to this showing (or describing) what character this is.
```suggestion
        content: '\203A'; // ›
```

---

Actually now that I think about it, the user might want to customize this so maybe we should use an icon? 🤔

---

Should we allow to customize it? Because this is our standard and users shouldn't use something else.

---

This code is removed now anyways, using a SVG instead of a CSS `content` symbol.